recipes-rubygems/rubygems-reek_6.0.6.bb

```diff
@@ -6,19 +6,18 @@ HOMEPAGE = "https://github.com/troessner/reek"
 LICENSE = "MIT"
 LIC_FILES_CHKSUM = "file://License.txt;md5=59252b93b9ae85dab91487d72990f77c"
 
-EXTRA_RDEPENDS += "\
+EXTRA_RDEPENDS_append += " \
     rubygems-codeclimate-engine-rb \
 "
 
 DEPENDS:class-native += "\
```

Comment:
potential issue: Multiple modifiers of append/prepend/remove/+= found in one operation